packages/deployments/contracts/contracts/lib/Connext/ConnextUtils.sol

```diff
@@ -158,4 +164,61 @@ library ConnextUtils {
     // Otherwise, swap to adopted asset
     return (pool.swapExact(_amount, _asset, adopted), adopted);
   }
+
+  /**
+   * @notice Called by relayer when they want to claim owed funds on a given domain
+   * @dev Domain should be the origin domain of all the transfer ids
+   * @param _domain - domain to claim funds on
+   * @param _recipient - address on origin chain to send claimed funds to
+   * @param _transferIds - transferIds to claim
+   * @param _relayerFeeRouter - The local nomad relayer fee router
+   * @param _transferRelayer - Mapping of transactionIds to relayer
+   */
+  function initiateClaim(
+    uint32 _domain,
+    address _recipient,
+    bytes32[] calldata _transferIds,
+    RelayerFeeRouter _relayerFeeRouter,
+    mapping(bytes32 => address) storage _transferRelayer
+  ) external {
```

Comment:
we should require the length of the transferIds is greater than 1 since thats included in the valid message length [here](https://github.com/connext/nxtp/blob/b71c852657fe62f24fa6e91f1dc050c0a177f9cc/packages/deployments/contracts/contracts/nomad-xapps/contracts/relayer-fee-router/RelayerFeeMessage.sol#L96-L100) and [here](https://github.com/connext/nxtp/blob/b71c852657fe62f24fa6e91f1dc050c0a177f9cc/packages/deployments/contracts/contracts/nomad-xapps/contracts/relayer-fee-router/RelayerFeeMessage.sol#L26)

Reply:
disregard included in the relayer fee router!